event_handler.go

```diff
@@ -41,6 +41,19 @@ func eventHandler(ctx *IrcContext, rtm *slack.RTM) {
 				if err != nil {
 					log.Printf("Error getting channel info for %v: %v", ev.Msg.Channel, err)
 					channame = "unknown"
+				} else if channel.IsMpIM {
+					channame = "&" + ev.Msg.Channel
```

Comment:
sorry for the confusion here, I meant we can use `@` for multi-person channels, and in a future PR we can use `&` for private channels

Reply:
Right, I read that wrong, will update the PR :+1:

Reply:
BTW `ev.Msg.Channel` is slack channel ID here because I noticed the channel name can become big. It's based on names of participants in it and some clients (I'm using weechat) don't support long channel names. Maybe you have better solution for this?

Reply:
Good point, I didn't think of long channel names because... with more than three people I create a channel. The channel ID is a good option, with no length or name collision issues. An alternative is to truncate the name, but this requires name collision handling. A simple way could be `prefix + ID + truncated_name + suffix`, where
* prefix is "@"
* ID is the slack ID (normally Cxxxx, Gxxxx, or Dxxxx)
* truncated name is the chan name as reported by the Slack API, truncated to X chars (e.g. 30? Just making numbers up)
* suffix is something that states that the name was truncated (e.g. "...", IIRC dots are allowed)

For example:

"@C123456-person1|person2|pers...."

Would that make sense?

Reply:
One thing about using `@` as a prefix is that it's also used for PMs, not sure if it's best idea to mix them?

Reply:
The `@` for users is only used for addressing them in messages, I think? Not for creating or referencing multi-user chats. Otherwise I agree with you that we should use another symbol

Reply:
I've used `&` as a prefix, as per RFC channel names can only be prefixed with a `&`, `#`, `+` or `!` : https://tools.ietf.org/html/rfc2812#section-1.3